Hw1/Hw1/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+using Hw1;
+
+while (true)
+{
+    Console.WriteLine("Please use zero numeration");
+    Console.Write("Would you like to Encode[E] or Decode[D]: ");
+    string answer = Console.ReadLine() ?? string.Empty;
+    if (answer == "E")
+    {
+        Console.Write("Input (Example: ABCDEF): ");
+        (string, int) output = BWT.Encode(Console.ReadLine() ?? string.Empty);
```

Comment:
```suggestion
        var output = BWT.Encode(Console.ReadLine() ?? string.Empty);
```
Здесь, например, с var сильно проще

А на следующей строке вместо Item1 и Item2 можно обращаться к EncodedData и Position, не зря же именовали :)